### PR TITLE
Add closing parenthesis at line 206 to fix syntax error occurring at line 208

### PR DESCRIPTION
**Pull request by raghavprasad13**

### CODE REVIEW

File: tests/microbenchmarks.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ def test_dict_set(test_case):
             dict["meow"] = 4
             dict["cockadoodledoo"] = 5
             dict["hiss"] = 6
-    """, timed=True)
+    """), timed=True)
 
 def test_class_init(test_case):
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 1: (E302) expected 2 blank lines, found 1

File: tests/microbenchmarks.py [hunk 2]
```diff
@@ -203,7 +203,7 @@ def test_dict_set(test_case):
...
     print("Running" , "test_class_init")
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 20: (E203) whitespace before ','

File: tests/microbenchmarks.py [hunk 3]
```diff
@@ -4,19 +4,24 @@
 
 from utils import *
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 1: (E402) module level import not at top of file

Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 1: (F403) 'from utils import *' used; unable to detect undefined names

File: tests/microbenchmarks.py [hunk 4]
```diff
@@ -4,19 +4,24 @@
...
 
+
 def test_small_integers(test_case):
     print("Running", "test_small_integers")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 5]
```diff
@@ -4,19 +4,24 @@
...
-        """), timed=True)
+        """),
+        timed=True)
+
 
 def test_booleans(test_case):
     print("Running", "test_booleans")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 6]
```diff
@@ -47,11 +52,14 @@ def test_booleans(test_case):
...
-        """), timed=True)
+        """),
+        timed=True)
+
 
 def test_global_var_load(test_case):
     print("Running", "test_global_var_load")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 7]
```diff
@@ -66,11 +74,14 @@ def test_global_var_load(test_case):
...
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def test_class_var_load(test_case):
     print("Running", "test_class_var_load")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 8]
```diff
@@ -111,11 +125,14 @@ def foo():
...
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def test_code(test_case):
     print("Running", "test_code")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 9]
```diff
@@ -136,11 +153,14 @@ def yarn(n):
...
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def test_cmp(test_case):
     print("Running", "test_cmp")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 10]
```diff
@@ -167,21 +187,27 @@ def test_cmp(test_case):
...
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def test_loops(test_case):
     print("Running", "test_loops")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 11]
```diff
@@ -167,21 +187,27 @@ def test_cmp(test_case):
...
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def test_dict_get(test_case):
     print("Running", "test_dictionary_get")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 12]
```diff
@@ -190,11 +216,14 @@ def test_dict_get(test_case):
...
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def test_dict_set(test_case):
     print("Running", "test_dictionary_set")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 13]
```diff
@@ -203,11 +232,14 @@ def test_dict_set(test_case):
...
+    """),
+        timed=True)
+
 
 def test_class_init(test_case):
-    print("Running" , "test_class_init")
-    test_case.runAsJava(adjust("""
+    print("Running", "test_class_init")
+    test_case.runAsJava(
+        adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 14]
```diff
@@ -242,11 +274,14 @@ class Z: pass
...
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def test_method(test_case):
     print("Running, test_method")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 15]
```diff
@@ -264,7 +299,9 @@ def C(self):
             obj.A()
             obj.B()
             obj.C()
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def main():
     test_case = TranspileTestCase()
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 17: (F405) 'TranspileTestCase' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 16]
```diff
@@ -4,6 +4,7 @@
 
 from utils import *
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 1: (E402) module level import not at top of file

Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 1: (F403) 'from utils import *' used; unable to detect undefined names

File: tests/microbenchmarks.py [hunk 17]
```diff
@@ -4,6 +4,7 @@
...
 
+
 def test_small_integers(test_case):
     print("Running", "test_small_integers")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 18]
```diff
@@ -14,6 +15,7 @@ def test_small_integers(test_case):
                     x = x + 1
         """), timed=True)
 
+
 def test_booleans(test_case):
     print("Running", "test_booleans")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 19]
```diff
@@ -49,6 +51,7 @@ def test_booleans(test_case):
             y = (False != False) != False
         """), timed=True)
 
+
 def test_global_var_load(test_case):
     print("Running", "test_global_var_load")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 20]
```diff
@@ -68,6 +71,7 @@ def test_global_var_load(test_case):
             print(y)
     """), timed=True)
 
+
 def test_class_var_load(test_case):
     print("Running", "test_class_var_load")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 21]
```diff
@@ -92,6 +96,7 @@ def speak(self):
 
     """), timed=True)
 
+
 def test_function_var_load(test_case):
     print("Running", "test_function_var_load")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 22]
```diff
@@ -113,6 +118,7 @@ def foo():
             foo()
     """), timed=True)
 
+
 def test_code(test_case):
     print("Running", "test_code")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 23]
```diff
@@ -138,6 +144,7 @@ def yarn(n):
             main(i)
     """), timed=True)
 
+
 def test_cmp(test_case):
     print("Running", "test_cmp")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 24]
```diff
@@ -169,6 +176,7 @@ def test_cmp(test_case):
                 x = [3.0] > [5.0]
     """), timed=True)
 
+
 def test_loops(test_case):
     print("Running", "test_loops")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 25]
```diff
@@ -179,6 +187,7 @@ def test_loops(test_case):
                         pass
     """), timed=True)
 
+
 def test_dict_get(test_case):
     print("Running", "test_dictionary_get")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 26]
```diff
@@ -192,6 +201,7 @@ def test_dict_get(test_case):
             dict.get("a")
     """), timed=True)
 
+
 def test_dict_set(test_case):
     print("Running", "test_dictionary_set")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 27]
```diff
@@ -203,10 +213,11 @@ def test_dict_set(test_case):
...
             dict["cockadoodledoo"] = 5
             dict["hiss"] = 6
-    """, timed=True)
+    """), timed=True)
+
 
 def test_class_init(test_case):
-    print("Running" , "test_class_init")
+    print("Running", "test_class_init")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 28]
```diff
@@ -244,6 +255,7 @@ class Z: pass
         g = 3.0
     """), timed=True)
 
+
 def test_method(test_case):
     print("Running, test_method")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 29]
```diff
@@ -266,6 +278,7 @@ def C(self):
             obj.C()
     """), timed=True)
 
+
 def main():
     test_case = TranspileTestCase()
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 17: (F405) 'TranspileTestCase' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 30]
```diff
@@ -1,8 +1,8 @@
+from utils import *
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 1: (F403) 'from utils import *' used; unable to detect undefined names

File: tests/microbenchmarks.py [hunk 31]
```diff
@@ -14,6 +14,7 @@ def test_small_integers(test_case):
                     x = x + 1
         """), timed=True)
 
+
 def test_booleans(test_case):
     print("Running", "test_booleans")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 32]
```diff
@@ -49,6 +50,7 @@ def test_booleans(test_case):
             y = (False != False) != False
         """), timed=True)
 
+
 def test_global_var_load(test_case):
     print("Running", "test_global_var_load")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 33]
```diff
@@ -68,6 +70,7 @@ def test_global_var_load(test_case):
             print(y)
     """), timed=True)
 
+
 def test_class_var_load(test_case):
     print("Running", "test_class_var_load")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 34]
```diff
@@ -92,6 +95,7 @@ def speak(self):
 
     """), timed=True)
 
+
 def test_function_var_load(test_case):
     print("Running", "test_function_var_load")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 35]
```diff
@@ -113,6 +117,7 @@ def foo():
             foo()
     """), timed=True)
 
+
 def test_code(test_case):
     print("Running", "test_code")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 36]
```diff
@@ -138,6 +143,7 @@ def yarn(n):
             main(i)
     """), timed=True)
 
+
 def test_cmp(test_case):
     print("Running", "test_cmp")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 37]
```diff
@@ -169,6 +175,7 @@ def test_cmp(test_case):
                 x = [3.0] > [5.0]
     """), timed=True)
 
+
 def test_loops(test_case):
     print("Running", "test_loops")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 38]
```diff
@@ -179,6 +186,7 @@ def test_loops(test_case):
                         pass
     """), timed=True)
 
+
 def test_dict_get(test_case):
     print("Running", "test_dictionary_get")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 39]
```diff
@@ -192,6 +200,7 @@ def test_dict_get(test_case):
             dict.get("a")
     """), timed=True)
 
+
 def test_dict_set(test_case):
     print("Running", "test_dictionary_set")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 40]
```diff
@@ -203,10 +212,11 @@ def test_dict_set(test_case):
...
             dict["cockadoodledoo"] = 5
             dict["hiss"] = 6
-    """, timed=True)
+    """), timed=True)
+
 
 def test_class_init(test_case):
-    print("Running" , "test_class_init")
+    print("Running", "test_class_init")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 41]
```diff
@@ -244,6 +254,7 @@ class Z: pass
         g = 3.0
     """), timed=True)
 
+
 def test_method(test_case):
     print("Running, test_method")
     test_case.runAsJava(adjust("""
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 42]
```diff
@@ -266,6 +277,7 @@ def C(self):
             obj.C()
     """), timed=True)
 
+
 def main():
     test_case = TranspileTestCase()
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 17: (F405) 'TranspileTestCase' may be undefined, or defined from star imports: utils

File: tests/microbenchmarks.py [hunk 43]
```diff
@@ -282,5 +294,6 @@ def main():
     test_class_init(test_case)
     test_method(test_case)
 
-if __name__== "__main__":
-  main()
+
+if __name__ == "__main__":
+	main()
```
Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 1: (W191) indentation contains tabs

Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 1: (E101) indentation contains mixed spaces and tabs

Comment (brutusthebee) [inline tests/microbenchmarks.py]: At column 8: (W292) no newline at end of file

**Comment by raghavprasad13** [comment]
Oh. Sorry, didn't notice that. Thank you for bringing it to my attention

**Pull request closed without merging**